compendium/ServiceComponent/test/TestCompInst_RefOrder.cpp

```diff
@@ -111,9 +123,10 @@ namespace
         ComponentInstanceImpl<TestServiceImpl1,
                               std::tuple<>,
                               std::true_type,
-                              ServiceDependency2,
-                              ServiceDependency1>
-            compInstance(binders); // compile error
+                              std::shared_ptr<ServiceDependency2>,
+                              std::vector<std::shared_ptr<ServiceDependency3>>,
+                              std::shared_ptr<ServiceDependency1>>
```

Comment:
this should be a separate test point, in addition to the one which already existed prior to this change.